### PR TITLE
[KOGITO-1232] Release 0.8.0-rc1 and CSV adjustments

### PR DESCRIPTION
**Pull request by ricardozanini**
See:
https://issues.redhat.com/browse/KOGITO-1232

Bump the version and some small changes on CSV to look nice.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: pkg/apis/app/v1alpha1/kogitodataindex_types.go [hunk 1]
```diff
@@ -89,11 +89,11 @@ type KogitoDataIndexStatus struct {
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 
 	// Status of the Data Index Service Deployment created and managed by it
-	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=true
+	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=false
```
Comment (radtriste) [inline pkg/apis/app/v1alpha1/kogitodataindex_types.go]: just for clarification, what is the purpose ?

Comment (ricardozanini) [reply pkg/apis/app/v1alpha1/kogitodataindex_types.go]: The `DeploymentSpec.Status` attribute doesn't play well with the CSV status display page, it's not rendered and the OCP console raises a NodeJS error. I'll change this status spec in the Service Definition Framework I'm working on.

**Comment by ricardozanini** [comment]
Tests have passed on Operator Hub. PR: https://github.com/operator-framework/community-operators/pull/1254

File: pkg/apis/app/v1alpha1/kogitodataindex_types.go (hunk 1 above)
Comment (ricardozanini) [reply pkg/apis/app/v1alpha1/kogitodataindex_types.go]: FYI: https://issues.redhat.com/browse/KOGITO-853

**Pull request merged**